pulsar-client/src/main/java/org/apache/pulsar/client/impl/auth/oauth2/AuthenticationDataOAuth2.java

```diff
@@ -44,7 +43,10 @@ public boolean hasDataForHttp() {
 
     @Override
     public Set<Map.Entry<String, String>> getHttpHeaders() {
-        return this.headers;
+        Map<String, String> headers = new HashMap<>();
+        headers.put(HTTP_HEADER_NAME, "Bearer " + accessToken);
+        headers.put(PULSAR_AUTH_METHOD_NAME, "token");
```

Comment:
This auth method name is already defined in `AuthenticationOAuth2#AUTH_METHOD_NAME`. What do you think about using that static reference?